package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "leaflet.pm",
-  "version": "2.2.0",
```

Comment:
Please do not update the version. It will be bumped when I release a new version of the library.

Reply:
Done